sdk/react/ooyalaSkinCore.js

```diff
@@ -121,7 +121,7 @@ OoyalaSkinCore.prototype.handlePress = function(n) {
       this.pushToOverlayStackAndMaybePause(OVERLAY_TYPES.DISCOVERY_SCREEN);
       break;
     case BUTTON_NAMES.AUDIO_AND_CC:
-      this.pushToOverlayStackAndMaybePause(OVERLAY_TYPES.AUDIO_AND_CC_SCREEN);
+      this.pushToOverlayStack(OVERLAY_TYPES.AUDIO_AND_CC_SCREEN);
       break;
```

Comment:
Shouldn't this change applied to OoyalaSkinCore.prototype.handleMoreOptionsButtonPress function also?

Reply:
In ticket there was nothing about more options button.

Reply:
That button (subtitles/multi-language) can be moved to MoreOption Panel dynamically. To move or not to move it to  that  button depends on current available size of skin layout and skin.json settings.

Reply:
done